src/databricksbundle/notebook/daipehelp/DaipeHelp.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import tomlkit
+
+from pathlib import Path
+
+
+class DaipeHelp:
+    def __init__(self, github_path, special_repos):
+        self.__github_path = github_path
+        self._special_repos = special_repos
+
+    def __find_module_path(self, module_name: str):
```

Comment:
why is `module_name` a string? it should be an instance of the class or a function